common.md

```diff
@@ -159,6 +159,7 @@ We represent Michelson values by constructors which wrap K primitive types.
   syntax Signature ::= #Signature(String)
   syntax OperationNonce ::= #Nonce(Int)
   syntax LambdaData ::= #Lambda(TypeName, TypeName, Block)
+  syntax Ticket ::= #Ticket(Data, Data, Data)
```

Comment:
Since we know that the first and third values of the ticket type are values of the `address` and `nat` types, lets use their internal representations of those types in our internal representation of `ticket`.

```suggestion
  syntax Ticket ::= #Ticket(Address, Data, Int)
```